### ISSUE TITLE
lxc-fedora doesn't work in 1.0.x anymore

### ISSUE THREAD

**Issue by daniel-baumann** [lxc/lxc#626]
it seems that the lxc-fedora template in 1.0.x doesn't work anymore with the current fedora releases, see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=792834

**Comment by ColinHuang** [comment]
#607

**Comment by a-detiste** [comment]
I haven't touched Redhat since 1999, have no idea what yum or DNF are except from the OSS world tabloid Phoronix...

This is halfway working now, but still fails at the end.

```diff
--- lxc-fedora  2015-08-18 14:17:00.000000000 +0200
+++ /usr/share/lxc/templates/lxc-fedora 2015-10-31 09:22:18.537333315 +0100
@@ -570,7 +570,7 @@
         # Right now, we are using Fedora 20 for the inial bootstrap.
         # We could make this the "current" Fedora rev (F > 15).
 
-        rsync -av mirrors.kernel.org::fedora/releases/20/Fedora/$basearch/os/LiveOS .
+        rsync -av mirrors.kernel.org::fedora/releases/21/Server/$basearch/os/LiveOS .
 
         if [[ 0 == $? ]]
         then
@@ -647,8 +647,8 @@
     # of this LiveOS image we're camped out on.  This is the beginning
     # of the butt ugly hack.  Look close or you may missing it...
 
-    rsync -av mirrors.kernel.org::fedora/releases/20/Fedora/$basearch/os/Packages/r/rpm-[0-9]* \
-        mirrors.kernel.org::fedora/releases/20/Fedora/$basearch/os/Packages/y/yum-[0-9]* .
+    rsync -av mirrors.kernel.org::fedora/releases/21/Server/$basearch/os/Packages/r/rpm-[0-9]* \
+        mirrors.kernel.org::fedora/releases/21/Server/$basearch/os/Packages/y/yum-[0-9]* .
 
     # And here it is...
     # The --nodeps is STUPID but F15 had a bogus dependency on RawHide?!?!
@@ -680,7 +680,7 @@
     echo "Stage 1 creation complete.  Building stage 2 Installation Bootstrap"
 
     mount -o bind ../bootstrap run/install
-    rsync -av mirrors.kernel.org::fedora/releases/20/Fedora/$basearch/os/Packages/f/fedora-release-20* .
+    rsync -av mirrors.kernel.org::fedora/releases/21/Server/$basearch/os/Packages/f/fedora-release-21* .
 
     # The --nodeps is STUPID but F15 had a bogus dependency on RawHide?!?!
     chroot . rpm --root /run/install --nodeps -ivh fedora-release-*
@@ -1288,8 +1288,8 @@
     if [ "$is_fedora" -a "$fedora_host_ver" ]; then
         release=$fedora_host_ver
     else
-        echo "This is not a fedora host and release missing, defaulting to 20 use -R|--release to specify release"
-        release=20
+        echo "This is not a fedora host and release missing, defaulting to 21 use -R|--release to specify release"
+        release=21
     fi
 fi
```


```
attention : fedora-release-21-2.noarch.rpm: Entête V3 RSA/SHA256 Signature, clé ID 95a43f54: NOKEY
attention : Unable to get systemd shutdown inhibition lock
Préparation...                       ################################# [100%]
Mise à jour / installation...
   1:fedora-release-21-2              ################################# [100%]
sed: impossible de lire ./run/install/etc/yum.repos.d/*: Aucun fichier ou dossier de ce type
Modules complémentaires chargés : fastestmirror, langpacks
Aucun dépôt n'est activé.
 Exécutez « yum repolist all » pour consulter la liste des dépôts installés.
 Vous pouvez activer des dépôts avec la commande yum-config-manager --enable <repo>
mount: mount point dev does not exist
mount: mount point proc does not exist
sed: impossible de lire ./etc/yum.repos.d/*: Aucun fichier ou dossier de ce type
chroot: failed to run command ‘yum’: No such file or directory
umount: /proc: target is busy
        (In some cases useful info about processes that
         use the device is found by lsof(8) or fuser(1).)
umount: dev: mountpoint not found

Build of Installation Bootstrap failed.  Temp directory
not removed so it can be investigated.
```

**Comment by nehaljwani** [comment]
I think commit f71e8f41fb129043323b13290c6697d27e603335 fixed this.

**Comment by evgeni** [comment]
this is fixed in 1.1.x

@stgraber / @brauner please close :)
